### PR TITLE
feat: display credentials per vaults in credentials screen

### LINKED ISSUE #1279: Update credentials tab to render all credentials from all vaults by default
When the user clicks on the `Credentials` link in the navbar or selects `All Vaults` from the vaults screen, we should display the following screen:
image

Currently, we just show all credentials, but do not split them up based on the vaults they are stored in

### LINKED ISSUE #1294: Provide actual vault name to credential details screen

### PR DESCRIPTION
**Pull request by birtony**
Fixes #1279

image
image

https://user-images.githubusercontent.com/33902374/144755189-bb99450f-c945-4db8-9629-a69568283b73.mov

Fixes #1294

https://user-images.githubusercontent.com/33902374/145117838-517a0e43-3bc8-4183-9c1e-10b5accd7b03.mov

Signed-off-by: Anton Biriukov <anton.biriukov@securekey.com>

### CODE REVIEW

File: scripts/wallet_web_dev_start.sh [hunk 1]
```diff
@@ -15,7 +15,7 @@ cd $ROOT/test/fixtures/wallet-web
 (source .env && docker-compose -f docker-compose-demo.yml -f docker-compose-server.yml -f docker-compose-web.yml down && docker-compose -f docker-compose-demo.yml -f docker-compose-server.yml -f docker-compose-web.yml up --force-recreate -d)
 
 echo "waiting for containers to start..."
-sleep 15s
+sleep 15
```
Comment (birtony) [inline scripts/wallet_web_dev_start.sh:18]: After upgrading macOS to Monterey, this started throwing an error. As per documentation, the numeric value provided as a parameter is always in seconds and does not require specifying the suffix `s`. Let me know if they works on older macOS versions, which I think it should...

image

File: cmd/wallet-web/src/pages/CredentialDetails.vue [hunk 2]
```diff
@@ -104,10 +107,44 @@ export default {
       toggleDelete,
     };
   },
-  computed: {
-    ...mapGetters(['getProcessedCredentialByID']),
-    credential() {
-      return this.getProcessedCredentialByID(this.$route.params.id);
+  data() {
+    return {
+      loading: true,
```
Comment (talwinder50) [inline cmd/wallet-web/src/pages/CredentialDetails.vue:113]: Where is this loading getting used in this file ?

File: test/ui-automation/test/specs/credential-interaction-flow.js [hunk 3]
```diff
@@ -468,21 +468,21 @@ describe("TrustBloc Wallet - Store/Share credential flow (CHAPI)", () => {
 
     await browser.navigateTo(`${browser.config.walletURL}/credentials`);
 
-    for (const [key, value] of credential.entries()) {
-      console.log("validate vc in wallet : start ", key);
+    // for (const [key, value] of credential.entries()) {
```
Comment (rolsonquadras) [inline test/ui-automation/test/specs/credential-interaction-flow.js:471]: instead of commenting these lines, we can add `it.skip` to line 466.

**Review by rolsonquadras — APPROVED** [review]

File: test/ui-automation/test/specs/credential-interaction-flow.js (hunk 3 above)
Comment (birtony) [reply test/ui-automation/test/specs/credential-interaction-flow.js:471]: We actually need line 469 to execute to redirect to the credentials screen in wallet

**Review by talwinder50 — CHANGES_REQUESTED** [review]

File: cmd/wallet-web/src/pages/CredentialDetails.vue (hunk 2 above)
Comment (birtony) [reply cmd/wallet-web/src/pages/CredentialDetails.vue:113]: We need to add the loading state in the UI which this variable will be used for as a condition

Comment (birtony) [reply cmd/wallet-web/src/pages/CredentialDetails.vue:113]: Filed https://github.com/trustbloc/wallet/issues/1295

**Review by talwinder50 — APPROVED** [review]

**Pull request merged**